code/modules/antagonists/heretic/status_effects/debuffs.dm

```diff
@@ -137,3 +137,48 @@
 			human_owner.adjustOrganLoss(ORGAN_SLOT_BRAIN, 20, 190)
 		if(95 to 100)
 			human_owner.adjust_confusion_up_to(12 SECONDS, 24 SECONDS)
+
+// STAR MARK
```

Comment:
Either replace this with a more explanatory documenting comment `///` or instead delete this, we can read the typepath.